page/index.html

```diff
@@ -7,6 +7,61 @@
     <title>Start - Myndigheten för yrkeshögskolan</title>
   </head>
   <body>
- 
+    <header class="grid-container">
+      <nav>
+        <ul>
+          <li>Start</li>
+          <li>Om oss</li>
+          <li>Verksamhetsområden</li>
+          <li>Statistik</li>
+          <li>Övright</li>
+        </ul>
+        <input type="text" placeholder="Ange sökord...">
+        <button>Sök</button>
+
+      </nav>
+      <div class="calender">
+        <div><h3>Kommande viktiga datum</h3>
+        </div>
+        <div><h5>December</h5>
+        <p>18: Ansök öppnar för att bedriva 
+          teckenspråks utbildning</p>
+          <p>19:Ansök öppnar för att bedriva 
+            Skrivtolkutbildning utbildning </p>
+            <p>25: MYH konferans för företagare</p>
+            <p>28: Sista ansökningsdagen till  
+              Konst- och kulturutbildning</p>
+      </div>
+      </div>
```

Comment:
Maybe this is an article instead of div, because it "encloses a block of related content that makes sense on its own without the rest of the page (e.g., a single blog post)."?

Reply:
Great feedback! I fix this issue.